src/lib/y2firewall/clients/firewall.rb

```diff
@@ -36,20 +34,23 @@ def initialize
         Yast.import "UI"
         Yast.import "Popup"
         Yast.import "PackageSystem"
+        Yast.import "CommandLine"
 
         textdomain "firewall"
       end
 
+      NOT_SUPPORTED = "YaST currently does not have a module for configuring" \
```

Comment:
this won;t work with translation. You have to use `N_` to mark it for markdown. Also some comment for translator would be nice ( like not translation stuff in quotes )